src/main/java/com/leomelonseeds/missilewars/listener/ArenaGameruleListener.java

```diff
@@ -545,7 +545,7 @@ public void onMove(PlayerMoveEvent event) {
                         ConfigUtils.sendConfigMessage("messages.poison", player, null, null);
                         player.addPotionEffect(new PotionEffect(PotionEffectType.POISON, 20 * 60 * 30, 1, false));
                     }
-                }, 10);
+                }, 20);
```

Comment:
```suggestion
                },10);
```

This doesn't make sense...
It would be more convenient to just delete the whole runTaskLater or to set the timer to half second if some waiting Is needed.

Reply:
The small delay is due to the poison height limit interfering with players jumping up at that height. With the recent addition of a jump boost passive, players with jump boost will still be interfered with which is annoying, hence the pull request.

Reply:
But than the suggestion is right to set this to a minimum delay of 10 ticks isn't it?

Reply:
Maybe check if the jump boost is activated on the player? But this shouldn't be done in this MR.
I would then accept my suggestion or close this MR while creating a new issue that describes the problem.